.secrets.baseline

```diff
@@ -1,5 +1,5 @@
 {
-  "version": "1.2.0",
+  "version": "1.3.0",
```

Comment:
Seems like the `.pre-commit-config.yaml` needs to be updated as well to `v1.3.0` to run it correctly locally.

Reply:
Updated in a333f0215663c042f70fea26a774362968a8faa2